test/bigdecimal/test_bigdecimal.rb

```diff
@@ -510,10 +510,10 @@ def test_round_up
 
     BigDecimal.mode(BigDecimal::ROUND_MODE, BigDecimal::ROUND_HALF_DOWN)
     assert_operator(n4, :>, n4 / 9 * 9)
-    assert_operator(n5, :>, n5 / 9 * 9)
+    assert_operator(n5, :<, n5 / 9 * 9)
```

Comment:
Half down rules are:
```
123.5 → 123 (half down)
123.50001 → 124 (round up because it's larger than half)
0.55550 → 0.555
0.55551 → 0.556
0.55555555.. → 0.556 (Because it's larger than half)
```

In master branch, rounding of `BigDecimal#/` was inconsistent.
```ruby
BigDecimal.mode(BigDecimal::ROUND_MODE, BigDecimal::ROUND_HALF_DOWN)

BigDecimal(10)/BigDecimal('18')
#=> 0.555555555555555555555555555555555555e0 (ends with 5)
BigDecimal(1)/BigDecimal('1.8')
#=> 0.555555555555555555555555555555555556e0 (ends with 6)
```